### PR TITLE
Issue 1046: Fixes issue with missing -M for user contrib server modules in external directory

### PR DESCRIPTION
**Pull request by glitch**
Issue 1046: Fixes issue with missing -M for user contrib server modules in external directory

Small fix to test for and add `-M` if missing to `ARKOUDA_SERVER_USER_MODULES`

### CODE REVIEW

**Comment by bmcdonald3** [comment]
See comments on other issue; there was a design discussion for not using `-M` and instead using the complete path to the module.

File: Makefile [hunk 1]
```diff
@@ -262,13 +262,19 @@ else
 	ARKOUDA_COMPAT_MODULES += -M $(ARKOUDA_SOURCE_DIR)/compat/lt-125
 endif
 
+ifdef ARKOUDA_SERVER_USER_MODULES
+	ifeq (,$(findstring \-M,$(ARKOUDA_SERVER_USER_MODULES)))
```
Comment (mhmerrill) [inline Makefile:266]: does this grock a list of directories contained in the `ARKOUDA_SERVER_USER_MODULES` env var?

Comment (mhmerrill) [reply Makefile:266]: so we can include modules from multiple directories.

**Comment by glitch** [comment]
I'm going to decline/close this one.  I think I understand @bmcdonald3 better now.  I mistook to the file path versus module name  and module directory path (I did the latter, set full path to the module directory and specified the module name in the cfg file, but I should have specified full path to the file containing the module.)  I'll add a wiki entry on this with an example to make it a bit more clear.

**Pull request closed without merging**